custom/icds_reports/templates/icds_reports/base_minimal.html

```diff
@@ -0,0 +1,42 @@
+{% load hq_shared_tags %}{% load i18n %}{% load statici18n %}<!DOCTYPE html>
+{% get_current_language as LANGUAGE_CODE %}
+<html lang="{{ LANGUAGE_CODE }}">
+  <head>
+    <title>
+      {% block title %}{% trans "CommCare HQ" %}{% endblock %}
```

Comment:
Didn't notice this earlier - ICDS templates typically don't reference CommCare, right?

Reply:
I see now it's overwritten in both child templates, just seems a bit strange.